projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/merchandising-carousel.ts

```diff
@@ -344,12 +344,13 @@ export function navigateToHomepage(): void {
 
 export function navigateToCategory(
   categoryName: string,
-  categoryCode: string
+  categoryCode: string,
+  force: boolean = true
```

Comment:
Could we revert this changes now that we don't reuse the functions from this file?

Reply:
hm yes i suppose so